sdk/test/unit/ens.test.ts

```diff
@@ -20,7 +21,7 @@ describe('ens', () => {
         {
           input: TEST_ENS_NAME,
           network: 'mainnet',
-          output: `0x${'0'.repeat(39)}1`,
+          output: ADDRESS_ONE,
```

Comment:
Please let's just define these locally (without export).

Reply:
Done 👍

Reply:
Do we need to `export const`?